patches/common/chromium/support_electron_default_error_mode_in_the_gpu_process.patch

```diff
@@ -0,0 +1,29 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Milan Burda <milan.burda@gmail.com>
+Date: Mon, 8 Apr 2019 20:08:20 +0200
+Subject: Support ELECTRON_DEFAULT_ERROR_MODE in the GPU process
+
+
+diff --git a/content/gpu/gpu_main.cc b/content/gpu/gpu_main.cc
+index 39e87967360a9c7a2856ce4df9c182c782efb2a5..586babaff18c1d034a9b53d238164e390ae1dce2 100644
+--- a/content/gpu/gpu_main.cc
++++ b/content/gpu/gpu_main.cc
+@@ -9,6 +9,7 @@
+ #include <utility>
+
+ #include "base/bind.h"
++#include "base/environment.h"
+ #include "base/lazy_instance.h"
+ #include "base/message_loop/message_loop.h"
+ #include "base/metrics/histogram_macros.h"
+@@ -225,6 +226,10 @@ int GpuMain(const MainFunctionParams& parameters) {
+       SEM_NOGPFAULTERRORBOX |
+       SEM_NOOPENFILEERRORBOX);
+
++  auto env = base::Environment::Create();
++  if (env->HasVar("ELECTRON_DEFAULT_ERROR_MODE"))
++    SetErrorMode(GetErrorMode() & ~SEM_NOGPFAULTERRORBOX);
```

Comment:
I don't love this; it's not upstreamable. Can we perhaps do this through a hook that's available somewhere in the GPU process startup path? If not, perhaps we could add such a hook?

Reply:
I will investigate that option

Reply:
Added a hook as suggested, can you please have another look?